### DISCUSSION TITLE
Is there a way to link to edit record form?

### DISCUSSION THREAD

**Discussion by adamshand** [pocketbase/pocketbase#2682] [Q&A] [answered]
Hello,

I'm playing with Pocketbase as the backend for a blog.  I'd like to use the PB admin interface for editing content.

Is there a way to link to the edit record screen in the admin interface?  

Is there a way to link to the new record screen in the admin interface?

It doesn't appear that this is possible, but just checking that I'm not missing something.

Thanks for the great project!
Adam.

**Comment by ganigeorgiev** [comment, marked as answer]
There is no direct link to the edit/update forms as they are implemented as overlay panels.

I've added to the [roadmap](https://github.com/orgs/pocketbase/projects/2/views/1) to consider adding a query parameter to trigger the the panel on page load (_we also will have to load the edit form record separately from the table listing_).

**Reply by adamshand** [reply]
Thanks for the prompt reply.  Looking forward to the possibility of a query parameter!  Cheers.

**Reply by ganigeorgiev** [reply]
_Just for info in case someone stumble here - support for linking to the record preview/update form was implemented in the `develop` branch together with some other minor UI improvements and will be available with the next v0.19.0 release (_there is no fixed ETA yet_)._

**Reply by adamshand** [reply]
Amazing, thank you!
